mage-os/mageos-magento2
# Update Branding

### Description (*)
Updates the admin footer and logo to Mage-OS branding.

### Manual Testing Scenarios (*)
1. Navigate to the admin login page
2. Observe that the Magento logo is replaced with the Mage-OS logo
---
1. Log into the admin
3. Observe that the logo has been updated to be the Mage-OS logo
4. Observe that the footer references Mage-OS
5. Observe that the copyright notice logo links to the Mage-OS website
6. Observe that the privacy policy and bug report links navigate to their Mage-OS counterparts

### Contribution checklist (*)
